buildSrc/src/main/kotlin/org/terasology/gradology/exec.kt

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.gradology
```

Comment:
I needed a package namespace for this stuff and kept getting stuck. Having a subdirectory named "gradle" seems like it would risk being confused with the "/gradle" directory that holds the wrapper in all gradle projects, and other words like "build" have similar problems